cinch/install-rhel7.yml

```diff
@@ -7,7 +7,7 @@
 # This playbook was tested with Ansible 1.8.4.
 
 - name: install cinch into a virtualenv on RHEL7
-  hosts: localhost
+  connection: local
```

Comment:
Indeed, this change actually causes the playbook to still pass Ansible lint, but causes it to fail ansible-playbook --syntax-check (this should probably be something that gets files as an ansible-lint bug!!). I'm preparing a PR that reverts this change and also adds checking this file to our lint.sh script.